data/json/items/vehicle/motors.json

```
@@ -19,7 +19,7 @@
     "id": "motor_enhanced",
     "name": { "str": "enhanced electric motor" },
     "description": "A very powerful and yet lightweight electric motor.  Useful for crafting.",
-    "weight": "100 kg",
+    "weight": "35 kg",
```

This is no bueno possibly this should have its materials adjusted, maybe add copper.
```suggestion
    "weight": "100 kg",
```